docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM docker.io/jupyter/scipy-notebook:lab-4.0.4
```

Comment:
This implicitly adds more requirements to `requirements.txt` that __could__ class.

Reply:
Yes, see

Everything in jupyter/minimal-notebook and its ancestor images
[altair](https://altair-viz.github.io/), [beautifulsoup4](https://www.crummy.com/software/BeautifulSoup/), [bokeh](https://docs.bokeh.org/en/latest/), [bottleneck](https://bottleneck.readthedocs.io/en/latest/), [cloudpickle](https://github.com/cloudpipe/cloudpickle), [conda-forge::blas=*=openblas](https://www.openblas.net/), [cython](https://cython.org/), [dask](https://www.dask.org/), [dill](https://pypi.org/project/dill/), [h5py](https://www.h5py.org/), [jupyterlab-git](https://github.com/jupyterlab/jupyterlab-git), [matplotlib-base](https://matplotlib.org/), [numba](https://numba.pydata.org/), [numexpr](https://github.com/pydata/numexpr), [openpyxl](https://openpyxl.readthedocs.io/en/stable/), [pandas](https://pandas.pydata.org/), [patsy](https://patsy.readthedocs.io/en/latest/), [protobuf](https://protobuf.dev/getting-started/pythontutorial/), [pytables](https://www.pytables.org/), [scikit-image](https://scikit-image.org/), [scikit-learn](https://scikit-learn.org/stable/), [scipy](https://scipy.org/), [seaborn](https://seaborn.pydata.org/), [sqlalchemy](https://www.sqlalchemy.org/), [statsmodel](https://www.statsmodels.org/stable/index.html), [sympy](https://www.sympy.org/en/index.html), [widgetsnbextension](https://ipywidgets.readthedocs.io/en/latest/user_install.html#installing-in-classic-jupyter-notebook), [xlrd](https://www.python-excel.org/) packages
[ipympl](https://github.com/matplotlib/ipympl) and [ipywidgets](https://ipywidgets.readthedocs.io/en/stable/) for interactive visualizations and plots in Python notebooks
[Facets](https://github.com/PAIR-code/facets) for visualizing machine learning datasets

Reply:
you could replace the spicy-notebook by the minimal-notebook. Smaller image and no unwanted packages